install/disconnected_install.adoc

```diff
@@ -184,6 +184,9 @@ Replace `<tag>` with the version to install. For example, specify `{latest-tag}`
 for the latest version. You can specify a different minor version.
 endif::[]
 +
+If you are using a containerized installation, pull
+`registry.redhat.io/openshift3/ose-ansible` in addition to these required images:
```

Comment:
Should specify an exact major release tag for ose-ansible, such as: registry.redhat.io/openshift3/ose-ansible:v3.11